OneCommunityGlobal/HGNRest
# show badges in weekly report

# Description
Jae: Make most difficult to earn badges show up by a person’s name on Reports Page
There are certain badges that are very hard to earn. When a person earns these, I’d like them to post to the right of the person’s other details on the Reports → Weekly Summaries Reports page. This is so that Management can see these badges were earned by that person and acknowledge them for it
- Make visible to anyone who has access to the page
- Make it remain on whatever weekly tab it is awarded on, just like a weekly summary stays on that tab and can be referenced up to two weeks past.
- Make it so multiple badges can be shown on this page if they happen to be earned in the same week
- Integrate this into the Other Links → Badge Management → Badge Development page. Add a column for “Reports Page Notification”. Checking a box here makes that badge one that shows up on the Weekly Summaries Reports page tab for that week when earned

## Related PRS (if any):
This backend PR is related to the [#1116 frontend PR](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1116).

## Main changes explained:
- create an attribute 'showReport' to the badge schema
- include badge data in the summary

## How to test:
1. check into current branch
2. do `npm run build` and `npm start` to run this PR locally
3. run the frontend PR
